modules/ROOT/pages/platform/api/authentication.adoc

```diff
@@ -0,0 +1,121 @@
+[[aura-api-authentication]]
+= Authentication
+:description: This page describes how to authenticate requests to the Aura API.
+
+label:AuraDB-Enterprise[]
+label:AuraDS-Enterprise[]
+
+[TIP]
+====
+The Aura API is currently in Public Beta for Aura Enterprise customers only. If you wish to participate in the Public Beta, please contact your Customer Success Manager.
+====
+
+The Aura API uses OAuth 2.0 for API authentication.
+
+== Creating credentials
+
+. Navigate to the https://console.neo4j.io/[Neo4j Aura Console Account Details page] in your browser.
```

Comment:
This should be https://console.neo4j.io/#account

Reply:
good spot